lib/steps.js

```diff
@@ -705,6 +705,13 @@ Then(/I get the(?: page's)?(?: JSON)? var(?:iable)?s and val(?:ue)?s/i, { timeou
   );
 });
 
+Then(/I expect the baseline PDF "([^"]+)" and the new PDF "([^"]+)" to be the same/i, async (existing_pdf_path, new_pdf_path) => {
```

Comment:
Is it worth making some of this optional so there can be a shorthand? I'm open to either. It would be something like the below (tested at regex101.com):

```suggestion
Then(/I expect(?: the baseline PDF)? "([^"]+)" and(?: the new PDF)? "([^"]+)" to be the same/i, async (existing_pdf_path, new_pdf_path) => {
```

The shortened form is `I expect "simple-doc-Baseline.pdf" and "simple-doc.pdf" to be the same`

Reply:
That is what I had at first, but since we treat the first argument differently from the second (the first is always searched for in the `sources` folder, and the second is always from the downloads), I wanted it to be unambiguous. Adding that shorthand makes it too easy for  people to write code that will confuse them later IMO.

If we did want this short hand, we'd need to let either argument be source or download, and the complexity of letting people use either argument for either PDF (but ensuring that one one from the source PDFs and the other is from the downloads, i.e. people aren't accidentally comparing two source PDFs in a test) isn't worth it.